src/components/Modals/OnchainIdentityModal.js

```diff
@@ -46,7 +50,10 @@ function OnchainIdentityModal({
 
```

Comment:
Specify `mode` in `useForm` call: https://react-hook-form.com/docs/useform#mode
I suggest `all`, as that's what we use in other parts of app.

Reply:
(btw. it's one line above the comment, it's just that GitHub doesn't show enough context.